settings.py

```diff
@@ -29,7 +29,7 @@
 }
 RECEIVE_TIMEOUT = 15 * 60  # s
 MANAGER_LOOP_TIMEOUT = 0.1  # s
-AGENT_CREATION_SLEEP = 0.15  # s
+AGENT_CREATION_SLEEP = 0.  # s
```

Comment:
`0.` XD?

Reply:
Może też warto by było tego nie pchać na release, tylko zostawić jakiś interwał typu 0.1s, gdyby ktoś miał to odpalać pierwszy raz

Reply:
Słusznie, zapomniałem xd